lib/downloader.go

```diff
@@ -45,6 +45,10 @@ func (d *Dkenv) DownloadDocker(version string) error {
 		return err
 	}
 
+	if http.DetectContentType(body) != "application/octet-stream" {
+		log.Fatal("Download not Content-Type application/octet-stream; check the version exists")
```

Comment:
Should return an error instead. Logging fatally here will prevent this from being testable.

Also, I think if it doesn't get a 200 it implies that the target isn't there - maybe update the error to say something like "content type mismatch" ?